lib/PrintAsClang/PrintClangFunction.cpp

```diff
@@ -713,6 +713,18 @@ ClangRepresentation DeclAndTypeClangFunctionPrinter::printFunctionSignature(
     if (kind == FunctionSignatureKind::CxxInlineThunk)
       ClangSyntaxPrinter(os).printGenericSignature(Signature);
   }
+  if (const auto *enumDecl = FD->getDeclContext()->getSelfEnumDecl())
+  {
```

Comment:
Minor: the Swift code style assumes that the brace goes on the same line as the condition